student/pom.xml

```diff
@@ -22,14 +22,16 @@
 	</licenses>
 
 	<properties>
-		<progman-client.version>3.0.0</progman-client.version>
+		<progman-client.version>3.0.1</progman-client.version>
```

Comment:
I had to bump the progman-client version to allow defining the PROGMAN_BASEURI property through environment variables rather than just java system properties (-Dprogman.baseuri=blah)  The property can still be defined through java system properties if preferred, however.